src/eval.c

```diff
@@ -675,6 +676,8 @@ void scriptCommand(client *c) {
 "    Kill the currently executing Lua script.",
 "LOAD <script>",
 "    Load a script into the scripts cache without executing it.",
+"DUMP <sha1> [<sha1> ...]",
+"    Dump a script from the scipts cache.",
```

Comment:
Does this need to be variadic (in that it takes multiple arguments)? You could always just send a multi-exec of individual `SCRIPT DUMP <sha1>` commands. Making a command not-variadic usually makes it easier for users to understand and easier to add more arguments in the future.